README.md

```diff
@@ -104,6 +112,14 @@ By default smartbanner would reappear if site path changes. Following example wo
 <meta name="smartbanner:hide-path" content="/">
 ```
 
+### Set "smartbanner" as global variable
+
+By default smartbanner not support manual construction. You can enable it and use "window.smartbanner" variable
+
+```html
+<meta name="smartbanner:api" content="true">
+```
+
```

Comment:
Whenever API is used, publishing should also be a responsibility of a developer which would make `start-on-load` obsolete. I'd support single option here, the API.

Reply:
Removed obsolete `start-on-load`